PharmacyInformationSystem/Pharmacy/Service/EmailService.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Mail;
+using System.Text;
+using System.Threading.Tasks;
+using PharmacyClassLib.Model;
+using PharmacyClassLib.Repository;
+using PharmacyClassLib.Repository.PharmacyOfferRepository;
+using PharmacyClassLib.Repository.RegistratedHospitalRepository;
+using PharmacyClassLib.Repository.TenderingRepository;
+
+namespace PharmacyClassLib.Service
+{
+    public class EmailService
+    {
+        private readonly NetworkCredential emailCredentials;
+        private readonly SmtpClient smtp;
+        private readonly IRegisteredHospitalRepository registeredHospitalRepository;
+        private readonly IPharmacyRepository pharmacyRepository;
+        private readonly IMedicationRepository medicationRepository;
+
+        public EmailService(IRegisteredHospitalRepository registeredHospitalRepository, IPharmacyRepository pharmacyRepository, IMedicationRepository medicationRepository)
+        {
+            this.registeredHospitalRepository = registeredHospitalRepository;
+            this.pharmacyRepository = pharmacyRepository;
+            this.medicationRepository = medicationRepository;
+
+            emailCredentials = new NetworkCredential("psworganisation8@outlook.com", "pswFtnOrganisation8");
```

Comment:
Need to replace email and password with env variables.
You can put default values for email and password.
But it is more flexible with env variables and sonar cloud code analysis will be successfull